node-require/calculate.js

```diff
@@ -0,0 +1,20 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+if (process.argv[3] === 'plus') {
+  console.log('result:', add(process.argv[2], process.argv[4]));
+}
+
+if (process.argv[3] === 'minus') {
+  console.log('result:', subtract(process.argv[2], process.argv[4]));
+}
+
+if (process.argv[3] === 'times') {
+  console.log('result:', multiply(process.argv[2], process.argv[4]));
+}
+
+if (process.argv[3] === 'over') {
+  console.log('result:', divide(process.argv[2], process.argv[4]));
+}
```

Comment:
Instead of accessing `process.argv` in several places it is better to store those values into variables first then use them where you need them.
```JAVASCRIPT
const num1 = Number(process.argv[2]);
const op = process.argv[3];
const num2 = Number(process.argv[4]);
```

Also converting them to numbers right away would prevent you from needing to use `Number` in all of your functions.